youtube/templates/watch.html

```diff
@@ -678,6 +678,20 @@ <h2 class="title">{{ title }}</h2>
 
     <script src="/youtube.com/static/js/av-merge.js"></script>
     <script src="/youtube.com/static/js/watch.js"></script>
+    <script>
+    var vido = document.getElementById('cuurent-vid'),
+    frameTime = 1 / 30;
```

Comment:
Different videos on YouTube have different framerates (e.g. 60 fps)

Reply:
> I suggest looking on stackoverflow for a more standard way to advance by one frame, if it exists. If not, you'll need to get the framerate of the selected format from the other code in youtube local
Thats the stackoverflow answer

Reply:
It could be fixed getting the fps from the selector?